src/app/sharedUI/screens/Artwork/Artwork.tsx

```diff
@@ -20,12 +40,104 @@ export const Artwork = () => {
     ),
   }))
 
+  const bottomSheetModalRef = useRef<BottomSheetModal>(null)
+
+  const snapPoints = useMemo(() => ["25%", "65%"], [])
+
+  const handlePresentModalPress = useCallback(() => {
+    bottomSheetModalRef.current?.present()
+  }, [])
+
+  const handleSheetChanges = useCallback((index: number) => {}, [])
+
+  const isEditArtworkHidden = GlobalStore.useAppState(
+    (state) => state.presentationMode.hiddenItems.editArtwork
+  )
+
   return (
-    <>
-      <Header safeAreaInsets />
+    <BottomSheetModalProvider>
+      <Header
+        safeAreaInsets
+        rightElements={
+          <Touchable onPress={handlePresentModalPress}>
+            <MoreIcon />
+          </Touchable>
+        }
+      />
       <Flex flex={1}>
         <ScrollableScreensView screens={screens} initialScreenName={params.slug} />
       </Flex>
-    </>
+      <BottomSheetModal
+        backdropComponent={() => (
+          <Touchable
+            style={{
+              opacity: 0.4,
+              width: "100%",
+              height: "100%",
+              position: "absolute",
+            }}
+            underlayColor="transparent"
+            onPress={() => bottomSheetModalRef.current?.close()}
+          >
+            <Flex background="black" height="100%" />
+          </Touchable>
+        )}
+        enablePanDownToClose
+        ref={bottomSheetModalRef}
+        index={1}
+        snapPoints={snapPoints}
+        onChange={handleSheetChanges}
+        handleComponent={() => (
+          <Flex position="relative" alignItems="center" justifyContent="center" height="50">
+            <Flex m={2} position="absolute" left="0">
+              <Touchable
+                onPress={() => bottomSheetModalRef.current?.close()}
+                underlayColor="transparent"
+              >
+                <CloseIcon />
+              </Touchable>
+            </Flex>
+            <Text weight="medium">More</Text>
+          </Flex>
+        )}
+      >
+        <Separator mb={2} />
+        <Flex alignItems="center" m={2}>
+          <ModalRow Icon={<ArtworkIcon />} label="View in Room" navigateTo={() => {}} />
+          <ModalRow Icon={<EditIcon />} label="Send by Email" navigateTo={() => {}} />
+          <ModalRow
+            Icon={<BriefcaseIcon />}
+            label="Add to Album"
+            navigateTo={() => navigation.navigate("AddArtworkToAlbum", { slug: params.slug })}
+          />
+          {!isEditArtworkHidden && <EditArtworkInCms slug={params.slug} />}
+        </Flex>
+      </BottomSheetModal>
+    </BottomSheetModalProvider>
+  )
+}
+
+interface ModalRowProps {
+  Icon: ReactElement
+  label: string
+  navigateTo: () => void
+}
+
+const ModalRow = ({ Icon, label, navigateTo }: ModalRowProps) => {
+  return (
+    <Touchable onPress={navigateTo} underlayColor="transparent">
+      <Flex flexDirection="row">
+        <Flex width="10%" alignItems="center" justifyContent="center">
```

Comment:
Lets avoid adding percentages value. 
```suggestion
        <Flex px={1} alignItems="center" justifyContent="center">
```

Reply:
these two mean very different things. what's the part of the design for this? let's see which one we want.

Reply:
I just tried to imitate the design which looks like this https://www.figma.com/proto/BQVn5aOJ4varTAgtz8mnai/Folio-2022?page-id=1312%3A3966&node-id=1649%3A11168&viewport=-1879%2C2704%2C0.36&scaling=scale-down&starting-point-node-id=1649%3A10878&show-proto-sidebar=1

Reply:
What is the problem with % values? 🤔

Reply:
I remember that we like to avoid having places in the app where we have padding to be 20 pixels, then somewhere else where it’s 35pilxels (for a phone that’s 350pixels and with a padding of 10%). We want users to get used to having the same padding everywhere across the app.

Reply:
I just like to follow the same approach. I would prefer using padding and margin values for FLex instead of width and height.

Reply:
that could be a good discussion for practice

Reply:
How about we keep the 10% width so that we can assign 10% to the first element, 80% to the second and 10% to the third, but remove horizontal "center" and add a padding to the left and the right? Would that be ok?

Reply:
ah now i see the figma link. ok so basically you had 10% of the width on the icon. yea i agree with sam, there is no need for a percentage here. the icon is a specific size, so we should use that as the width and height for this flex. using a percentage will mean that it looks different on wider and narrower phones/ipads, and thats not what we want.